db/schema.rb

```diff
@@ -46,7 +46,7 @@
     t.text "body"
     t.string "url"
     t.datetime "created_at"
-    t.datetime "updated_at"
+    t.datetime "uodated_at"
```

Comment:
🧐 this seems to be a typo. How did this happen in the schema file?

Reply:
There is an old migration file with this typo and because I executed all migrations on my computer I get this typo. How can we fix this?

Reply:
@zaziemo The affected migration file create_blog_posts was changed 10 month ago and thereby the error came to the file. I have no idea if the database on heroku has now this typo inside and why this change was done.

Reply:
@zaziemo I checked out that the databases on staging and production have the correct spelling "updated". So, should I correct the typo in the migration file and in the schema?

Reply:
yes, this would be great. "Old" databases wouldn't run this migration and we prevent new ones from adding the error.